### PR TITLE
feat: ability to add custom project name

### PR DESCRIPTION
**Pull request by acheronfail**
As discussed in #77.

### CODE REVIEW

File: project_manager.sublime-settings [hunk 1]
```diff
@@ -12,6 +12,11 @@
...
 
-    // The string to append to indicate an active project.
-    "active_project_indicator": "*"
+    // The string to use as the active indicator for the project list.
+    "active_project_indicator": "*",
+
+    // How the projects in the project list should be formatted. Supported variables
+    // are "project_name" and "active_project_indicator". For inactive projects
+    // "active_project_indicator" will be an empty string.
+    "project_display_format": "{project_name} - {active_project_indicator} - "
```
[the remark this answers is not in the document]
Comment (acheronfail) [reply project_manager.sublime-settings]: Sorry, that's my bad - I had that left over from testing. Fixed!

File: project_manager.py [hunk 2]
```diff
@@ -198,13 +198,15 @@ def which_project_dir(self, pfile):
 
     def render_display_item(self, item):
         project_name, info = item
+        active_project_indicator = str(self.settings.get('active_project_indicator', '*'))
+        display_format = str(self.settings.get('project_display_format', '{project_name} {active_project_indicator}'))
```
[the remark this answers is not in the document]
Comment (acheronfail) [reply project_manager.py]: 👍

File: project_manager.sublime-settings [hunk 3]
```diff
@@ -12,6 +12,11 @@
...
 
-    // The string to append to indicate an active project.
-    "active_project_indicator": "*"
+    // The string to use as the active indicator for the project list.
+    "active_project_indicator": "*",
+
+    // How the projects in the project list should be formatted. Supported variables
+    // are "project_name" and "active_project_indicator". For inactive projects
+    // "active_project_indicator" will be an empty string.
+    "project_display_format": "{project_name}{active_project_indicator} - "
```
Comment (randy3k) [inline project_manager.sublime-settings]: 😕 still, you didn't remove the trailing ` - `.

Comment (acheronfail) [reply project_manager.sublime-settings]: Ah 🤦‍♂️ sorry, I edited the wrong file - I had two folders checked out, one to test and my fork!

**Comment by randy3k** [comment]
Thank you. It is what I got now.

screen shot 2018-06-20 at 5 42 48 am

**Comment by acheronfail** [comment]
Awesome! Using emoji is a great idea!

**Comment by acheronfail** [comment]
Want to make a release so it updates on PackageControl? :)

**Comment by randy3k** [comment]
I tagged a release already. It may take a while for PackageControl to cache the update.

**Pull request merged**